### PR TITLE
test connectors content

### PR DESCRIPTION
**Pull request by lauramthomas**
### Description
Added Test Connectors content to API Testing -> Integrations -> PagerDuty and Webhook Connectors

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing/integrations/pagerduty-webhooks.md [hunk 1]
```diff
@@ -67,6 +67,25 @@ Like the PagerDuty Connector, setting up an outgoing webbook connector allows Sa
...
+
+To test a connector:
+1. Log in to Sauce Labs > click **API Testing**.
+2. Click on any project.
+3. Within that project, click **Settings** and then click **Connector**.
+4. On the **Connectors** page, click **Test Connectors**.
+<img src={useBaseUrl('img/api-testing/test-connectors-nav.png')} alt="Test Connectors button"/>
+5. In the **Connectors Test Drive** window, select either the **Failure** or **Success** radio button, and then click **Trigger**.
+  * Failure - You will receive a notification in all connector platforms in your list
+  * Success - If the **On_success** toggle is set to **Ture**, you will receive a notification for the connector
```
Comment (LoryApiFortress) [inline docs/api-testing/integrations/pagerduty-webhooks.md:81]: @lauramthomas Typo: True not "Ture"

File: docs/api-testing/integrations/pagerduty-webhooks.md [hunk 2]
```diff
@@ -67,6 +67,25 @@ Like the PagerDuty Connector, setting up an outgoing webbook connector allows Sa
...
+<img src={useBaseUrl('img/api-testing/trigger-confirmation.png')} alt="Test Connectors dialog box"/>
+
+:::note
+To check the functionality of connectors like PagerDuty that trigger an incident upon failure, and then resolve that incident, you need to first test for **Failure** and then for **Success**.
```
Comment (LoryApiFortress) [inline docs/api-testing/integrations/pagerduty-webhooks.md:85]: @lauramthomas "To check the functionality of connectors like PagerDuty that trigger an incident upon failure, and then resolve that incident **with the following success**,..."

File: docs/api-testing/integrations/pagerduty-webhooks.md [hunk 3]
```diff
@@ -67,6 +67,25 @@ Like the PagerDuty Connector, setting up an outgoing webbook connector allows Sa
 ### Incoming Webhooks
 See [Adding Incoming Webhooks](/api-testing/integrations/apifctl-cicd-integration/#adding-incoming-webhooks).
 
+## Testing Connectors
+The Test Connectors feature tests the functionality of each connector as defined in its settings, but will not display any notifications on the dashboard. Testing a connector before executing it can minimize the time you spend creating connectors.
+
+To test a connector:
```
Comment (LoryApiFortress) [inline docs/api-testing/integrations/pagerduty-webhooks.md:73]: @lauramthomas This feature tests all the connectors you have set up with one click so it is better to use the plural

Comment (lauramthomas) [reply docs/api-testing/integrations/pagerduty-webhooks.md:73]: Fixed

File: docs/api-testing/integrations/pagerduty-webhooks.md (hunk 2 above)
Comment (lauramthomas) [reply docs/api-testing/integrations/pagerduty-webhooks.md]: Fixed

**Review by lauramthomas — COMMENTED** [review]
@LoryApiFortress I made your suggested changes. This is good to go!

**Review by LoryApiFortress — APPROVED** [review]

**Pull request merged**